webapp/components/product-info.js

```diff
@@ -37,7 +38,7 @@ const versionPatterns = Object.freeze({
 });
 
 // The set of all browsers known to the wpt.fyi UI.
-const AllBrowserNames = Object.freeze(['chrome', 'edge', 'firefox', 'safari', 'servo', 'webkitgtk']);
+const AllBrowserNames = Object.freeze(['chrome', 'deno', 'edge', 'firefox', 'safari', 'servo', 'webkitgtk']);
```

Comment:
```suggestion
const AllBrowserNames = Object.freeze(['chrome', 'edge', 'firefox', 'safari', 'servo', 'webkitgtk', 'deno']);
```

Reply:
I thought they were alphabetical :shrug:

Reply:
Yea you might be right. I will change the tests instead